### PR TITLE
Add setOrderProfile function to context

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What problem is this solving?
Adds a new `setOrderProfile` function to the context.

[Related clubhouse story](https://app.clubhouse.io/vtex/story/32809/extrair-l%C3%B3gica-de-profile-do-checkout-identification).

#### How should this be manually tested?
[Workspace](https://lucas--checkoutio.myvtex.com/checkout/identification)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

**Review by marcoskwkm — APPROVED** [review]

**Comment by lucasecdb** [comment]
@marcoskwkm I updated the types of the `setOrderProfile` function to accept more than just the user `email`, might be worth reviewing again

**Review by marcoskwkm — APPROVED** [review]

File: react/package.json [hunk 1]
```diff
@@ -3,13 +3,20 @@
...
   "dependencies": {
     "react": "^16.13.0",
     "react-apollo": "^3.1.3"
   },
   "devDependencies": {
     "@types/react": "^16.9.23",
     "@types/react-dom": "^16.9.5",
-    "vtex.render-runtime": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.render-runtime@8.93.2/public/@types/vtex.render-runtime"
+    "@vtex/test-tools": "^3.0.3",
+    "vtex.checkout-graphql": "https://lucas--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.checkout-graphql@0.24.0+build1583340197/public/@types/vtex.checkout-graphql",
```
Comment (jeffersontuc) [inline react/package.json]: I think you might need to unlink `checkout-graphql` and `checkout-resources` before running `vtex setup`.

**Review by jeffersontuc — APPROVED** [review]

File: react/package.json (hunk 1 above)
Comment (lucasecdb) [reply react/package.json]: I will!

Comment (lucasecdb) [reply react/package.json]: done :-)

**Pull request merged**